src/index.js

```diff
@@ -1,20 +1,38 @@
-import _ from 'lodash';
+// import _ from 'lodash';
```

Comment:
- Please, let's remove unused code form our codebase to make our code clean and maintainable.

Reply:
### PACKAGE.JSON


- Hey, Your app is not running because you've not added the `start and build` to the script, this makes your app not run successfully. please, kindly add the following:
- You'd have to install the ` 'webpack-dev-server'` to ensure it works correctly.


```
"scripts": {
    "linters": "npx hint . && npm run run-stylelint && npx eslint .",
    "run-stylelint": "stylelint \"**/*.css\"",
    "build": "webpack",
    "start": "webpack serve --open"
  },
```

Reply:
### webpack config file-

- Please, let's add the mode we want our app to run, either production or development. see the image to see where to add it below:

![Screenshot from 2023-01-18 02-22-08](https://user-images.githubusercontent.com/59397159/213057774-4baba766-5296-4272-ba53-f874ab8e63e4.png)

Reply:
### README FILE

- Hey, please update the readme file as some content is not pointing to this current project.
- let's also remove all the placeholder text as they're there to guide you on how to add content to the readme file and are not meant to be there after updating the readme template to suit your project needs.
---

:point_right: *You can choose to add some content here or remove this section to make your readme more professional*
![Screenshot from 2023-01-18 02-31-25](https://user-images.githubusercontent.com/59397159/213059125-3fa609af-9e69-4bc0-97a0-b259bbcd3e39.png)

:point_right: ***Let's kindly add some features to this project***
![Screenshot from 2023-01-18 02-31-03](https://user-images.githubusercontent.com/59397159/213059128-66436eb6-f19f-4aa3-bbc4-2ccb868b604a.png)

:point_right: *You've worked on this project alone, kindly update this author section*
![Screenshot from 2023-01-18 02-30-52](https://user-images.githubusercontent.com/59397159/213059131-5c9fb743-8581-4d0d-a0b4-df6a937c13d4.png)

:point_right: ***Let's kindly add some instructions on how users can have this project locally on their machine and update the remaining header section in the image below***
![Screenshot from 2023-01-18 02-30-18](https://user-images.githubusercontent.com/59397159/213059134-ea35c199-a4a6-424a-8e40-5dc7a4da282f.png)

:point_right: *Let's add the key features about this project to help users know what the project is all about*
![Screenshot from 2023-01-18 02-29-23](https://user-images.githubusercontent.com/59397159/213059141-e01a4eb3-82e0-4f86-8b09-9ba58e1e3ad4.png)

:point_right: *Kindly update the tech stack to the list of technologies you've used in building this project*
![Screenshot from 2023-01-18 02-29-13](https://user-images.githubusercontent.com/59397159/213059144-8330977f-7a25-45e6-863d-1df15a8bff6d.png)


- Let's add a license as the link is now broken to make the readme more professional.